hazelcast/src/main/java/com/hazelcast/map/impl/operation/ClearBackupOperation.java

```diff
@@ -16,11 +16,13 @@
 
 package com.hazelcast.map.impl.operation;
 
-import com.hazelcast.nio.serialization.DataSerializable;
+import com.hazelcast.map.impl.MapDataSerializerHook;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
 import com.hazelcast.spi.BackupOperation;
 import com.hazelcast.spi.impl.MutatingOperation;
 
-public class ClearBackupOperation extends MapOperation implements BackupOperation, MutatingOperation, DataSerializable {
```

Comment:
Isn't it easier to impose it on MapOperation? This way you know you are going to catch all map operations.